frontend_app_workspace_advanced/webpack.tracim.config.js

```diff
@@ -1,2 +1,2 @@
 module.exports = require('./webpack.config.js')
-module.exports.externals = { tracim_frontend_lib: 'tracim_frontend_lib.lib', ...require('tracim_frontend_vendors/dist/externals.json') }
+module.exports.externals = { tracim_frontend_lib: 'tracim_frontend_lib.lib', ...Object.fromEntries(Object.keys(require('tracim_frontend_vendors')).map(dep => [dep, "tracim_frontend_vendors['" + dep + "']"])) }
```

Comment:
this line is too long, it would be better to have line breaks or to put the intelligence in a variable before the line

you could also use object literals in the .map() to avoid string concatenation

Reply:
note that file names have changed in next commit. Explaining the "outdated" tag